packages/components/src/components/table/table-head-cell/table-head-cell-styles.ts

```diff
@@ -28,39 +25,28 @@ export const getComponentCss = (
     '@global': {
       ':host': addImportantToEachRule({
         display: 'table-cell',
-        padding: `${pxToRemWithUnit(2)} ${pxToRemWithUnit(12)} ${pxToRemWithUnit(8)}`,
-        borderBottom: `1px solid ${contrastMediumColor}`,
+        padding: `2px ${spacingFluidSmall} ${spacingFluidSmall} ${spacingFluidSmall}`,
```

Comment:
4th value can be removed since it's identical to 2nd one.

Reply:
Fixed it